driver.sh

```diff
@@ -19,17 +19,56 @@ setup_variables() {
   # Turn on debug mode after parameters in case -h was specified
   set -x
 
+  # torvalds/linux is the default repo if nothing is specified
+  case ${REPO:=linux} in
+    "linux")
+      owner=torvalds ;;
+    "linux-next")
+      owner=next
+      tree=linux-next ;;
+    "4.4"|"4.9"|"4.14"|"4.19")
+      owner=stable
+      branch=linux-${REPO}.y ;;
+  esac
+  url=git://git.kernel.org/pub/scm/linux/kernel/git/${owner}/${tree:=linux}.git
+
   # arm64 is the current default if nothing is specified
   case ${ARCH:=arm64} in
-    "arm")
+    "arm32_v5")
+      config=multi_v5_defconfig
+      image_name=zImage
+      qemu="qemu-system-arm"
+      qemu_ram=512m
+      qemu_cmdline=( -machine palmetto-bmc
+                     -no-reboot
+                     -dtb "${REPO}/arch/arm/boot/dts/aspeed-bmc-opp-palmetto.dtb"
```

Comment:
Changing `${REPO}` to `${tree}` here will be a little more future proof in case we start building these architectures on LTS branches: https://github.com/ClangBuiltLinux/continuous-integration/commit/7c4b607a5d36d155edff01df18e3f949df4b2c88

Reply:
Okay. I added setting of tree in the switch statement, please take a look.